_docs/_latest/use/config.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Config
+order: 007
+---
+
+# {{ page.title }}
+
+Forseti Config is a tool for you to set the configuration of the CLI.
+
+You can learn more about the client-server architecture of Forseti [here]({% link _docs/latest/concepts/architecture.md %}).
+
+## Running Forseti Config
+
+#### Show the current local configuration
+
+```bash
+$ forseti config show
+```
+
+The command above will output the current local configuration. 
+
+#### Resetting the local configuration
+
+```bash
+$ forseti config reset
+```
+
+The command above will reset local configuration back to it's original state.
+
+#### Formatting the CLI output
+
+```bash
+$ forseti config format <FORMAT>
+```
+
+The command above will update the output format of the CLI to `<FORMAT>`.
+
+Possible values for `<FORMAT>` are `text` or `json`.
```

Comment:
See my comment above about matching the format of `docs/latest/configure/notifier/index.md` and the way it describes keys and their values.

Reply:
Updated